### ISSUE TITLE
This app supports H sites?

### ISSUE THREAD

**Issue by ElvisTekencio01** [Creckeryop/NOBORU#135]
I'm sorry if I offended someone or i broke any rule.

**Comment by diotodd** [comment]
Yes it does, you can find the ones that are supported on the Catalogs tab. (If you disabled the NSFW filter)
